roles/vault_unseal/defaults/main.yml

```diff
@@ -1,7 +1,11 @@
 ---
 # Allow vault_vip_url and vault_vip_address for backwards compatibility.
-vault_vip_address: "{{ vault_vip_url | default('') }}"
-vault_api_addr: "{{ ('https://' ~ vault_vip_address ~ ':8200') if vault_vip_address else '' }}"
+vault_protocol: "{{ 'https' if vault_tls_key and vault_tls_cert else 'http' }}"
+vault_vip_address: "{{ vault_vip_url | default(vault_bind_address) }}"
+vault_api_addr: "{{ vault_protocol ~ '://' ~ vault_vip_address ~ ':8200' }}"
+vault_bind_address: "127.0.0.1"
+vault_tls_key: ""
+vault_tls_cert: ""
 
```

Comment:
I can see how this has ended up here, but it doesn't seem right. This role doesn't need to (and shouldn't) know the key & cert.

Reply:
IMO we should only have the variables here that are used in the role.

Reply:
Done, defaulted to http for the protocol in vault_unseal, but wondering if we should default to https everywhere and remove http - that's not really secure by default ;-)

Reply:
Default to https yes, remove http no - useful for testing?